File: new_server/vueblog-common/src/main/java/cn/smilex/vueblog/common/entity/blog/RequestBlog.java
package cn.smilex.vueblog.common.entity.blog;

import cn.smilex.vueblog.common.entity.tag.SelectBlogTag;
import com.fasterxml.jackson.annotation.JsonAlias;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author smilex
 * @date 2022/12/6/6:34
 * @since 1.0
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class RequestBlog {
    private Long id;

    @JsonAlias("user_id")
    private Long userId;

    @JsonAlias("sort_id")
    private Integer sortId;

    private String title;
    private String description;
    private String content;
    private List<SelectBlogTag> tag;
    private Short status;
    private String key;

    @JsonAlias("key_title")
    private String keyTitle;
}

File: new_server/vueblog-common/src/main/java/vip/smilex/vueblog/common/entity/blog/RequestBlog.kt
package vip.smilex.vueblog.common.entity.blog

import vip.smilex.vueblog.common.entity.tag.SelectBlogTag
import com.fasterxml.jackson.annotation.JsonProperty

/**
 * @author smilex
 * @date 2023/12/4 21:56:03
 */
class RequestBlog {
    var id: Long? = null

    @JsonProperty("user_id")
    var userId: Long? = null

    @JsonProperty("sort_id")
    var sortId: Int? = null

    var title: String? = null
    var description: String? = null
    var content: String? = null
    var tag: List<vip.smilex.vueblog.common.entity.tag.SelectBlogTag>? = null
    var status: Short? = null
    var key: String? = null

    @JsonProperty("key_title")
    var keyTitle: String? = null
}